packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -160,57 +176,63 @@ const RecentDiscussionSubscribeReminder = ({classes}: {
       </AnalyticsInViewTracker>
     </AnalyticsContext>
   }
-  
+
   if (loading) {
-    return <div>
+    return <div className={classes.root}>
       <Loading/>
     </div>
   } else if (subscriptionConfirmed) {
     return <AnalyticsWrapper branch="already-subscribed">
-      You are subscribed to the best posts of LessWrong!
+      <div className={classes.message}>
+        <CheckRounded className={classes.checkIcon} />
+        You are subscribed to the best posts of {forumType}!
+      </div>
     </AnalyticsWrapper>
   } else if (!currentUser || adminBranch===0) {
     // Not logged in. Show a create-account form and a brief pitch.
     return <AnalyticsWrapper branch="logged-out">
       <div className={classes.message}>
+        <MailOutline className={classes.mailIcon} />
         To get the best posts emailed to you, create an account! {subscriptionDescription}
```

Comment:
Hmm personally I subscribed to the digest email and have never voted... but I actually I went with Ben's suggestion to make this capsule look the same in multiple cases (logged out, unsubscribed from all emails, and otherwise not subscribed to the digest)

Reply:
I think I disagree with Ben here, but will concede the point